beacon-chain/execution/service.go

```diff
@@ -754,6 +754,10 @@ func (s *Service) initializeEth1Data(ctx context.Context, eth1DataInDB *ethpb.ET
 			}
 		}
 	} else {
+		if eth1DataInDB.Trie == nil && eth1DataInDB.DepositSnapshot != nil {
```

Comment:
error messages should contain punctuation, so it might be better to use a hyphen instead of a comma

Reply:
This error will stop the node, so I think it is fine to use punctuation in this case. If you still feel strongly I can change it, but the sentence structure is off then